omero/sysadmins/unix/install-web/web-deployment.txt

```diff
@@ -311,9 +274,6 @@ For more details refer to
 EXPERIMENTAL: Sync workers
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-.. note:: Experimental configurations are not ready for production use.
```

Comment:
As above, not sure about leaving the 'Experimental' title but removing the note about what that means.